.github/monitor-typescript-errors/utils/airtable.js

```diff
@@ -0,0 +1,38 @@
+const axios = require( 'axios' ).default;
+
+const BASE_URL = 'https://api.airtable.com/v0';
+const TABLE_ID = 'appIIlxUVxOks06sZ';
+const API_KEY = process.env[ 'AIRTABLE_API_KEY' ];
+
+const TABLE_NAME = 'TypeScript Migration';
+const TYPESCRIPT_ERRORS_COLUMN_NAME = 'TypeScript Errors';
+const DATE_COLUMN_NAME = 'Date';
+
+// https://community.airtable.com/t/datetime-date-field-woes/32121
+const generateDateValueForAirtable = () => {
+	const today = new Date();
+	const string = today.toLocaleDateString();
+
+	return new Date( string );
+};
+
+exports.addRecord = async ( errorsNumber ) =>
+	axios.post(
```

Comment:
Yes, my idea is to break the CI because it is important that we monitor our progress. The error handling is done [here](https://github.com/woocommerce/woocommerce-blocks/blob/490fab10ad4092a5a705fa6b5a221613172311c2/.github/monitor-typescript-errors/index.js#L52-L58)